proxy/src/control_plane/client/cplane_proxy_v1.rs

```diff
@@ -296,6 +403,49 @@ impl super::ControlPlaneApi for NeonControlPlaneClient {
         Ok(Cached::new_uncached(auth_info.secret))
     }
 
+    async fn get_allowed_ips(
```

Comment:
Hmm. I wonder if it's better to, instead of making a new method that allows not using a context, create a background context that you can pass to get_allowed_ips_and_secret directly.